app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,6 @@
+class LikesController < ApplicationController
+  def create
+    Like.create(user: current_user, post_id: params[:post_id])
+    redirect_back(fallback_location: root_path)
+  end
```

Comment:
- [ ] It will be nice to have a conditional statement like `if/else`, that also takes care of the negative outcome. If creating likes fails, it should redirect us to a safe path.

Reply:
I don't understand what do you mean by a safe path where like doesn't even go anywhere it just reloads

Reply:
But here is my confusion I want to also do the same thing would it be considered good practice if I do if/else on same good?

```Ruby
if like.save
redirect_back(fallback_location: root_path)
else 
redirect_back(fallback_location: root_path)
end
```

Reply:
I took my chances and made it so user can't like a post twice 😄, Thanks for the great discussion!